src/extension.ts

```diff
@@ -62,7 +63,7 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
         registerGraphCommands();
         const codeLensProvider = registerMongoCommands(editorManager);
 
-        const accountContextValues: string[] = [GraphAccountTreeItem.contextValue, DocDBAccountTreeItem.contextValue, TableAccountTreeItem.contextValue, MongoAccountTreeItem.contextValue];
+        const accountContextValues: string[] = [GraphAccountTreeItem.contextValue, DocDBAccountTreeItem.contextValue, TableAccountTreeItem.contextValue, MongoAccountTreeItem.contextValue, PostgresServerTreeItem.contextValue];
```

Comment:
There's several commands here that postgres doesn't support. Suggest splitting it up like this:
```
const cosmosDBAccountContextValues: string[] = [GraphAccountTreeItem.contextValue, DocDBAccountTreeItem.contextValue, TableAccountTreeItem.contextValue, MongoAccountTreeItem.contextValue];
const allTopLevelContextValues: string[] = [...cosmosDBAccountContextValues, PostgresServerTreeItem.contextValue];
```

I'm not a huge fan of the variable names I came up with, but you get the idea. You should only pass `allTopLevelContextValues` to `showTreeItemPicker` for commands that postgres also supports - I believe only "Open in Portal".

For example, if you run "Cosmos DB: Delete Account" from the command palette, a postgres account should not show up